contentgrid-spring-data-rest/build.gradle

```diff
@@ -35,21 +35,30 @@ dependencies {
     implementation 'jakarta.servlet:jakarta.servlet-api'
 
     implementation 'org.springframework.data:spring-data-rest-webmvc'
+    implementation 'com.github.paulcwarren:spring-content-rest'
 
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.boot:spring-boot-starter-web'
     testImplementation 'jakarta.transaction:jakarta.transaction-api'
 
+
+
     testFixturesAnnotationProcessor 'com.querydsl:querydsl-apt::jpa'
     testFixturesAnnotationProcessor 'jakarta.persistence:jakarta.persistence-api'
 
     testFixturesImplementation platform(project(':contentgrid-spring-boot-platform'))
 
     testFixturesApi 'org.springframework.data:spring-data-jpa'
+    testFixturesApi 'com.github.paulcwarren:spring-content-commons'
+    testFixturesApi 'com.github.paulcwarren:spring-content-rest'
+
+
     testFixturesImplementation 'org.springframework.boot:spring-boot-starter-data-jpa'
     testFixturesImplementation 'org.springframework.boot:spring-boot-starter-data-rest'
-    testFixturesImplementation 'org.projectlombok:lombok'
+    testFixturesImplementation "com.github.paulcwarren:spring-content-rest-boot-starter"
+    testFixturesImplementation "com.github.paulcwarren:spring-content-fs-boot-starter"
 
+    testFixturesImplementation 'org.projectlombok:lombok'
```

Comment:
Lombok is a compile-time only annotation processor, so it should not be pulled in as an implementation dependency